packages/hoppscotch-sh-admin/src/pages/teams/index.vue

```diff
@@ -13,24 +13,57 @@
 
       <div class="overflow-x-auto">
         <div
-          v-if="fetching && !error && teamList.length === 0"
+          v-if="fetching && !error && teamsList.length === 0"
           class="flex justify-center"
         >
           <HoppSmartSpinner />
         </div>
 
         <div v-else-if="error">{{ t('teams.load_list_error') }}</div>
 
-        <TeamsTable
-          v-else
-          :teamList="teamList"
-          @goToTeamDetails="goToTeamDetails"
-          @deleteTeam="deleteTeam"
-          class=""
-        />
+        <div v-else-if="teamsList.length == 0" class="px-2">
+          <p class="text-lg">
+            {{ t('teams.no_teams') }}
+          </p>
+        </div>
+
+        <div v-else class="m-5">
```

Comment:
I am using a margin of 5 around the table because I want a bit of space on all sides of the table and I don't want it to reach the ends of the border. I feel it helps the table to stand out instead of spreading across the border of the frame.

Reply:
I don't think the table will hit the border since there is horizontal padding on both sides by the parent div. also its better to align with the button on top
![image](https://github.com/hoppscotch/hoppscotch/assets/53208152/366fdd4c-8111-476c-a203-705d9fb7fbc3)